### DISCUSSION TITLE
Claudie-Maude - Calculs pour son PhD

### DISCUSSION THREAD

**Discussion by Claudie-Maude** [Bureau-du-Forestier-en-chef/MoSiR#42] [Ideas]
Bonjour,

Voici mes enjeux : 
1) Ce serait chouette de pouvoir téléverser plusieurs .json en même temps comme intrant pour les calculs (j'en ai 60 à importer)
2) Lorsque je suis dans l'onglet de calcul, je ne peux pas modifier les entrées déjà ajoutées pour modifier en cas d'erreur ou pour garder, par exemple, le même type de sortie mais changer mon titre de fichier en sortie. Je dois à chaque fois refaire les manipulations (changer le temps de simulation, les noeuds en sortie, ajuster les facteurs de potentiel de réchauffement sur 100 ans, etc.).
3) Le bouton "Exécuter" est caché sous la longue liste du menu déroulant de mes noeuds en sortie. Je peux y accéder, mais c'est assez long comme manipulation si plusieurs simulations sont à faire.
4) mon problème d'itération semble s'être réglé de lui-même !

Merci !

**Comment by Claudie-Maude** [comment]
Aussi : 5. Je suis actuellement en mesure d'importer le choix du "output" précédent 1 fois sur 2. Cela veut dire que je dois entrer manuellement à nouveau le choix du "output" manuellement 1 fois sur 2. Or, dans mes travaux, je veux toujours le même output (horizon de temps, HWP, noeud, unité) mais pour différents inputs: La 1 fois sur 2 que je peux ré-importer/utiliser le choix du output précédent me sauve beaucoup de manipulation :)   

6. Ce serait chouette de pouvoir supprimer des fichier .json dans la section "télécharger"; cela me permettrait de mettre à jour en fonction de mon évolution, de retirer les fichiers .json en input et en output déjà traités et de téléverser d'autres fichiers sans avoir à réouvrir MoSIR pour rafraichir la liste de fichiers dans l'onglet "téléverser".

**Comment by Landry-G** [comment]
J'en prend bonne note, merci de tester et de faire des suggestions ! À titre informatif (je l'écris bientôt dans la doc donc c'est normal que tu ne le sache pas), lorsqu'on importe un graphe, la mémoire de tout ce qui est importé est vidée. Il y avait trop souvent des problèmes lorsqu'on importait plusieurs graphes, avec plusieurs inputs et reports différents. Il était donc plus simple de repartir de 0 si l'utilisateur téléverse ou importe de Miro un nouveau graphique, ce qui pourrait expliquer pourquoi tu perds ton report une fois sur deux! 

Sur cette note, ça sera effectivement utile de faire ton point 6 pour aider au point 5. 

Je met sur ma liste le point 1, 2 et 3 pour l'ergonomie d'utilisation!
